frontend/src/layout.css

```diff
@@ -0,0 +1,9 @@
+.body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  background-color: #000000;
```

Comment:
I believe this background color should be `#111111`.

Reply:
Okay.  I couldn't tell